plugin.video.ahltv/resources/lib/globals.py

```diff
@@ -58,9 +59,9 @@
     60014: "https://farm66.staticflickr.com/65535/49391830453_e964db6ab0_c.jpg", # Milwaukee
     60015: "https://www.ontarioreign.com/assets/img/Moulson-BAK-Web-413-68af5a4817.jpg", # Ontario
     60016: "https://pbs.twimg.com/media/EMHgfRmXYAUGjJg?format=jpg&name=small", # Providence
-    60017: "https://amerks.com/img/user/1947a071-25ce-4464-8c41-9de4d8b08741_20200110-RT.jpg", # Rochester
+    60017: "https://www.amerks.com/assets/img/202102010MV0103-a2f885f781.JPG", # Rochester
```

Comment:
would be nice if you could package all those images as a resource.addon and add it as a dependency to your addon. Using direct links to random images on the internet just makes the stability of your addon fragile.
Look for instance on how retrospect addon does it.